src/pages/install/index.md

```diff
@@ -4,25 +4,25 @@ import TabItem from '@theme/TabItem';
 
 # Installing RGBDS
 
-Most people will want to use a release (the latest the bestest!), but if you are willing to help us test new features, you may consider [using a development version](/install/master).
+Most people will want to use a release (the more recent the better), but if you are willing to help us test new features, you may consider [using a development version](/install/master).
 
 The two main options to install a release of RGBDS are to download pre-built executables, or to [build from source](/install/source).
 Below is a list of <abbr title="Operating System">OS</abbr>es for which we know pre-built executables are available.
 
 If your OS isn’t listed below, try finding `rgbds` in your package manager—refer to your OS' documentation for help.
 If you still can’t find RGBDS, or the specific version you are looking for is unavailable, you will have to [compile it from source](#building-from-source).
 
+
 <Tabs>
-<TabItem value="windows" label="Windows">
+<TabItem value="linux" label="Linux">
 
-The install instructions change a bit depending on the environment you wish to use RGBDS with.
+TO DO
```

Comment:
We should suggest, in order:

- Using the distribution's package manager, if available (providing some examples?)
- Using our pre-built binaries
- Docker?
- Compiling from source
- ??? Anything I missed?